views/homework/feedback.hbs

```diff
@@ -1,20 +1,20 @@
 <section id="feedback" class="tab-content section-homeworkdetails">
     <div class="homework print">
         {{#if @root.submission.grade}}
-            Du hast <b class="grade">{{@root.submission.grade}}%</b> richtig gelöst.
+           '{{$t "homework.text.youHave" }}' <b class="grade">{{@root.submission.grade}}%</b> '{{$t "homework.text.solvedCorrectly" }}'
```

Comment:
I Suggest leaving parts like these out for now and adding them to the code snippets on 
[our docs](https://docs.schul-cloud.org/pages/viewpage.action?pageId=113643606 )
as this way of dealing with variables will cause problems when translating to other languages.
We'll have to use some kind of interpolation here, i guess.